contracts/compliance/modular/modules/CountryRestrictModule.sol

```diff
@@ -153,21 +181,31 @@ contract CountryRestrictModule is AbstractModule {
      *  no transfer action required in this module
      */
     // solhint-disable-next-line no-empty-blocks
```

Comment:
For instance, having the function written accross multiple line lines makes the `// solhint-disable-next-line no-empty-blocks` ineffective. It should be put a bit below.